.env.test

```diff
@@ -73,7 +73,7 @@ L1_CORE_CONTRACT_ADDRESS="0xE2Bb56ee936fd6433DC0F6e7e3b8365C906AA057"
 
 ##### SNOS #####
 ## This is needed right now because Madara doesn't support getProof
-RPC_FOR_SNOS=""
+RPC_FOR_SNOS="http://81.16.176.130:9545"
```

Comment:
we need to remove this can't push this URL.

Reply:
removed.